src/hostnet/vmnet.ml

```diff
@@ -25,7 +25,7 @@ module Init = struct
 
   let default = {
     magic = "VMN3T";
-    version = 1l;
+    version = 22l;
```

Comment:
is 22 expected? shouldn't be just 2?

Reply:
Yes, it is to be higher than the protocol version used internally in Docker for Mac, which is currently 21.